src/components/Header/index.js

```diff
@@ -18,6 +18,7 @@ class Header extends React.Component {
 
   logout = () => {
     localStorage.removeItem('user');
+    return window.location.replace(ROUTES.home);
```

Comment:
@Tittoh Let's not use window just a normal `Link` from *react-router-dom* is enough. Sth like
`<Link to 'ROUTES.home' />`

Reply:
This method updated the header only without redirecting, so I passed props and pushed history.